src/envo/envo-edit.owl

```diff
@@ -40880,7 +40880,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501168> <http://purl.obolibrar
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_03501169> (pediatric unit facility)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/ENVO_>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501169> "A hospital unit facility which is used to provide pediatric medicine."@en)
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501169> "A hospital unit facility which is used to provide pediatric medicine."@en)
```

Comment:
@ddooley Similar to above: "pediatric services"?

Reply:
ditto!